rbac/management/utils.py

```diff
@@ -199,6 +199,14 @@ def validate_uuid(uuid, key="UUID Validation"):
         raise serializers.ValidationError({key: _(message)})
 
 
+def validate_group_name(name):
+    """Verify name provided is valid."""
+    if name in ["Custom default access", "Default access"]:
```

Comment:
I think we'll want to make this case-insensitive, as I can still create `custom default access` or `default access` with this.

Reply:
Fixed 👍